recipes/tedna/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: tedna
+  version: "1.2.2"
+
+build:
+  number: 0
+
+source:
+  fn: tedna_1.2.2.tar.gz
+  url: https://urgi.versailles.inra.fr/content/download/3481/29402/file/tedna_1.2.2.tar.gz
+  md5: b60b9b974e349db692eea75aad15b5e9
+
+requirements:
+  build:
+  - google-sparsehash
```

Comment:
Dear @cmonjeau, can you add `bc` in the build dep list?
(it will solve the `/bin/sh: bc: command not found error`, maybe not the following one laule)